docker/modules/Dockerfile.networking

```diff
@@ -33,4 +33,9 @@ COPY docker/include/security/nginxfail.conf /root/nginx/
 COPY docker/include/security/nginx-site /var/www/nginx-site
 COPY docker/include/security/tlsfaux tlsfaux/
 
+COPY subset/dot1x/freeradius/users /etc/freeradius/3.0/users
```

Comment:
mmm... need to think about this. There shouldn't be a dependency from the core stuff on anything in subset/.  LIkewise, there shouldn't be a dependency on anything in the docker/ on things in subset.  What we might need to do is uplevel the common stuff to daq/misc/dot1x or something -- similar to how daq/usi is comomn between a few things -- really that should be likewise in something like daq/misc/usi (but that can be moved separately)

Reply:
Pushed freeradius to misc/